structurerandom.m

```diff
@@ -43,7 +39,7 @@ function structurerandom(x1,x2,y1,y2)
         S.SE(i).length = lengte(S.SE(i).u,S.SE(i).v);
         
         cubed(1:Nterm) = radius^3; %determining radius extra element using murray's law: r0^3 = r1^3+r2^3
-        S.IE(nie+counterie).radius = (S.IE(nie+counterie).radius^3 - sum(cubed))^(1/3); 
+        S.IE(nie+counterie).radius = (S.IE(nie).radius^3 - sum(cubed))^(1/3); 

```

Comment:
yes I know that is not the solution....this morning i've understood it.
About lines deleted I was wrong using github,sorry.
I didn't want.